src/libsystemd-network/test-ipv4ll.c

```diff
@@ -168,6 +170,13 @@ static void test_basic_request(sd_event *e) {
 
                 sd_event_run(e, UINT64_MAX);
                 assert_se(basic_request_handler_bind == 1);
+
+                if (start_address->s_addr != 0) {
+                        struct in_addr address;
+
+                        assert_se(sd_ipv4ll_get_address(ll, &address) == 0);
```

Comment:
Not a big matter, but we usually use `>= 0` instead of `== 0`, even if the function currently returns `0` on success.